app/build.gradle.kts

```diff
@@ -142,7 +143,7 @@ android {
     /** these config values override those in AndroidManifest.xml.  Can also set versionCode and versionName */
     defaultConfig {
         applicationId = applicationIdStandard
-        minSdk = 23
+        minSdk = 26
```

Comment:
What requires this?

Reply:
Not sure, I'll check what breaks if I put it back

Reply:
```
MethodHandle.invoke and MethodHandle.invokeExact are only supported starting with Android O (--min-api 26): Lorg/tartarus/snowball/SnowballProgram;find_among([Lorg/tartarus/snowball/Among;I)I
```

Reply:
Lucene requires this: 
<img width="414" alt="image" src="https://github.com/user-attachments/assets/1cc191f3-920f-4f62-b1b0-861412988dae">

Reply:
About 2% of our users are using Android versions 6 / 7 / 7.1.